.github/workflows/ci_cron.yml

```diff
@@ -328,3 +328,18 @@ jobs:
       - name: Annotate errors
         if: ${{ failure() }}
         uses: ./.github/actions/annotate
+
+  gem_notifications:
+    runs-on: ubuntu-20.04
+    env:
+      gems: "activerecord bunny dalli delayed_job excon http httpclient mongo puma sidekiq sinatra tilt rack rails rake redis resque unicorn"
+    steps:
+      - uses: ruby/setup-ruby@v1.90.0
+        with:
+          ruby-version: 2.7
```

Comment:
I'm curious, why did you decide to use Ruby 2.7?

Reply:
I went with ruby 2.7 because it was the version we use for the rubocop and simplecov jobs. I just liked the consistency of it, but happy to change it if you think that is a better move.

Reply:
This consistency with the other jobs will be helpful when it comes time to switch to Ruby 3.1 as part of the migration to Ubuntu 22. 👍